api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ConceptTranslatorImpl.java

```diff
@@ -28,30 +30,33 @@
 import org.openmrs.ConceptMapType;
 import org.openmrs.ConceptReferenceTerm;
 import org.openmrs.ConceptSource;
+import org.openmrs.Duration;
+import org.openmrs.module.fhir2.FhirConstants;
 import org.openmrs.module.fhir2.api.FhirConceptService;
 import org.openmrs.module.fhir2.api.FhirConceptSourceService;
 import org.openmrs.module.fhir2.api.translators.ConceptTranslator;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
+import org.openmrs.module.fhir2.model.FhirConceptSource;
+import org.springframework.cache.annotation.Cacheable;
 
 @Slf4j
-@Component
 public class ConceptTranslatorImpl implements ConceptTranslator {
```

Comment:
Could you please add implements OpenmrsService to ConceptTranslator for consistency?